### ISSUE TITLE
Spurious test failure: Mouse won't qdel

### ISSUE THREAD

**Issue by JohnFulpWillard** [tgstation/tgstation#68963]
```
 ## REF SEARCH Found /mob/living/simple_animal/mouse [0x30000bd] in list Datums -> /datum/spatial_grid_cell [0x2100765d] -> hearing_contents (list).
## REF SEARCH Completed search for references to a /mob/living/simple_animal/mouse.
## TESTING: GC: -- [0x30000bd] | /mob/living/simple_animal/mouse was unable to be GC'd --
Error: /mob/living/simple_animal/mouse hard deleted 1 times out of a total del count of 22
FAIL: /datum/unit_test/create_and_destroy 520s
	REASON #1: /mob/living/simple_animal/mouse hard deleted 1 times out of a total del count of 22 at code/modules/unit_tests/create_and_destroy.dm:167
```

https://github.com/tgstation/tgstation/actions/runs/2791010137/attempts/1

**Comment by LemonInTheDark** [comment]
We've been seeing this since spatial grid. I hate it here

**Comment by LemonInTheDark** [comment]
kyler did we ever get the hint code into unit tests?

**Comment by Kylerace** [comment]
@LemonInTheDark no not yet

**Comment by JohnFulpWillard** [comment]
> We've been seeing this since spatial grid. I hate it here

You think Kapu's spatial grid PR fixed this? Don't think I've seen it in a while.

**Comment by LemonInTheDark** [comment]
maybeee, @Kylerace ?

**Comment by Kylerace** [comment]
@LemonInTheDark @JohnFulpWillard canonical movement should have fixed it

**Comment by LemonInTheDark** [comment]
alllright bet closed by #82085
